sdk/servicebus/service-bus/src/serviceBusClient.ts

```diff
@@ -42,34 +42,38 @@ export class ServiceBusClient {
   constructor(connectionString: string, options?: ServiceBusClientOptions);
   /**
    *
-   * @param host The hostname of your Azure Service Bus.
-   * @param tokenCredential A valid TokenCredential for Service Bus or a
-   * Service Bus entity.
-   * @param options Options for the service bus client.
+   * @param fullyQualifiedNamespace The full namespace of your Service Bus instance which is 
```

Comment:
> The full namespace of your Service Bus instance

Sounds weird?

Reply:
Right now it mirrors Event Hubs, if there is better text, I want both places to be updated